src/main/java/org/ballerinalang/containers/docker/cmd/DockerCmd.java

```diff
@@ -261,4 +283,33 @@ private void printMainImageSuccessMessage(String imageName) {
         outStream.println("\tdocker run --name " + containerName + " -it " + imageName);
         outStream.println();
     }
+
+    /*
+    Print helpful messages for functionality to perform after image build for a Ballerina bal/balx main/service files.
+     */
+    private void printImageSuccessMessage(String imageName) {
```

Comment:
Same as above, shouldn't other similar methods be removed now?

Reply:
+1